lib/brainstem/query_strategies/filter_and_search.rb

```diff
@@ -2,24 +2,34 @@ module Brainstem
   module QueryStrategies
     class FilterAndSearch < BaseStrategy
       def execute(scope)
-        ordered_search_ids = run_search(scope, filter_includes.map(&:name))
-        scope = scope.where(id: ordered_search_ids)
+        scope, ordered_search_ids = run_search(scope, filter_includes.map(&:name))
         scope = @options[:primary_presenter].apply_filters_to_scope(scope, @options[:params], @options)
-        scope = @options[:primary_presenter].apply_ordering_to_scope(scope, @options[:params])
-        count = scope.count
-        scope = paginate(scope)
-        primary_models = evaluate_scope(scope)
+
+        if ordering?
+          count = scope.count
+          scope = paginate(scope)
+          scope = @options[:primary_presenter].apply_ordering_to_scope(scope, @options[:params])
+          primary_models = evaluate_scope(scope)
+        else
+          filtered_ids = scope.pluck(:id)
+          count = filtered_ids.size
+
+          ordered_ids = order_for_search(filtered_ids, ordered_search_ids, false)
+          ordered_paginated_ids = paginate_array(ordered_ids)
+
+          scope = scope.unscoped.where(id: ordered_paginated_ids)
+          primary_models = scope.to_a
```

Comment:
I recommend leaving a comment here as to why we're _not_ using `evaluate_scope` in this case